sensors/ev3_uart_sensor_ld.c

```diff
@@ -232,12 +232,12 @@ struct ev3_uart_port_data {
 	u8 requested_mode;
 	u8 type_id;
 	u8 new_mode;
-	u32 raw_min;
-	u32 raw_max;
+	s32 raw_min;
+	s32 raw_max;
 	u32 pct_min;
```

Comment:
wait a minute. In this file, we are using u32 here because these are actually 32-bit floating point values coming from the sensors. Since the kernel doesn't have a floating point type, this is actually more like "raw data", so don't change anything in this file.